ga4gh/backends.py

```diff
@@ -365,18 +365,23 @@ class Backend(object):
     """
     Superclass of GA4GH protocol backends.
     """
-    def __init__(self, dataDir, Dataset):
+
+    def __init__(self, dataDir, VariantSet):
```

Comment:
I find this a bit confusing all right; what about `variantSetClass` to make the type explicit?